File: src/bench1.swift
var iter = 10000000;
var sum = 0;

var x: Array<Int> = [];
var y: Array<Int> = [];

for (var i=0; i<iter; i++) {
  x.append(i)
  y.append(i + 1)
}

for (var i=0; i<iter; i++) {
  if (i % 5 == 0 && x[i] % 2 == 0) {
    sum += x[i];
  }
}

for (var i=0; i<iter; i++) {
  if (i % 5 == 0 && y[i] % 2 == 0) {
    sum += y[i];
  }
}

print(sum)

File: src/bench1.rs
fn main() {
    let iter = 10000000;
    let mut sum = 0;

    let mut x: Vec<i64> = Vec::new();
    let mut y: Vec<i64> = Vec::new();

    for i in 0..iter {
      x.push(i);
      y.push(i + 1);
    }

    for i in 0..iter {
      if i % 5 == 0 && x[i as usize] % 2 == 0 {
        sum += x[i as usize]
      }
    }

    for i in 0..iter {
      if i % 5 == 0 && y[i as usize] % 2 == 0 {
        sum += y[i as usize]
      }
    }

    println!("{}",sum);
}
